### ISSUE TITLE
gpg_signature: Should not be a tuple

### ISSUE THREAD

**Issue by stuhood** [pypa/warehouse#3059]
Sometime in the last two weeks, gpg-signed uploads to https://upload.pypi.org/legacy/ began to break with the following error message:
```
Uploading distributions to https://upload.pypi.org/legacy/
Signing ${snip}.whl

You need a passphrase to unlock the secret key for
user: "${snip}"
2048-bit RSA key, ID ${snip}, created ${snip}

Uploading ${snip}.whl
HTTPError: 400 Client Error: gpg_signature: Should not be a tuple. for url: https://upload.pypi.org/legacy/
```

This seems potentially related to #3049 ?

This is in a venv using twine for the upload and the following dependency versions. 
```
$ twine-deps.venv/bin/twine --version
twine version 1.9.1 (pkginfo: 1.4.1, requests: 2.18.4, setuptools: 38.5.1,
requests-toolbelt: 0.8.0, tqdm: 4.19.5)

$ twine-deps.venv/bin/python --version
Python 2.7.14
```
Neither our invoke of `twine`, nor the dependencies have changed in the last two weeks.

**Comment by di** [comment]
Thanks for the report @stuhood, and sorry for the inconvenience. This is fixed in #3061 and will be deployed momentarily.

**Comment by stuhood** [comment]
Huzzah, thanks! Confirmed working.
